src/dotnet/APIView/APIViewWeb/Models/SamplesRevisionUploadModel.cs

```diff
@@ -1,12 +1,12 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 
 namespace APIViewWeb.Pages.Assemblies
 {
-    public class UsageSampleUploadModel
+    public class SamplesRevisionUploadModel
```

Comment:
Shouldn't this be in LeanModel namespace?

Reply:
I does not matter. AM only using the LeanModel namespace to group models I recently added, just for my own sanity. I will eventually move everything back into Models.